exercises/word-count/word-count.spec.js

```diff
@@ -29,12 +29,12 @@ describe('words()', () => {
   });
 
   xit('respects case', () => {
```

Comment:
Absolutely! Thanks for catching that and please change it if it's not too
much trouble.
On Mon, Apr 4, 2016 at 3:56 PM Rubén Chavarría <notifications@github.com>
wrote:

> In exercises/word-count/word-count.spec.js
> <https://github.com/exercism/xecmascript/pull/210#discussion_r58437285>:
>
> > @@ -29,12 +29,12 @@ describe('words()', () => {
> >    });
> >
> >    xit('respects case', () => {
>
> Same as xjavascript [image: :smile:]
> I think this description does not reflect what the spec does now. What
> about normalizes to lower case? @matthewmorgan
> <https://github.com/matthewmorgan>, if you agree, I can do the change
> while merging the PR.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/exercism/xecmascript/pull/210/files/555e16d9508f63c3dc6205e620d8087550c0a717#r58437285>
>